### PR TITLE
Rainbow kit integration

### PR DESCRIPTION
**Pull request by R-iskey**

### CODE REVIEW

File: src/screens/subscribe/components/ConnectWalletButtons.tsx [hunk 1]
```diff
@@ -1,65 +1,27 @@
...
-    setSelectedWallet(connector.id);
 
-    if (connector.id === 'metaMask' && isMobile()) {
-      if (connector.ready) {
-        return connect({ connector });
-      }
-
-      window.open(`https://metamask.app.link/dapp/${window.location.href}`, '_blank');
-      return;
+  useEffect(() => {
```
Comment (nivo33) [inline src/screens/subscribe/components/ConnectWalletButtons.tsx:16]: Why like this in the useEffect? no handler for onClicked or something?

File: src/screens/subscribe/components/ConnectWalletButtons.tsx [hunk 2]
```diff
@@ -1,65 +1,27 @@
...
-          <Button
-            disabled={isLoading}
-            key={connector.id}
-            onClick={() => connectWallet(connector)}
-            width={300}
-            isLoading={isLoading && selectedWallet === connector.id}
-          >
-            {connector.name}
-          </Button>
-        ))}
```
Comment (nivo33) [inline src/screens/subscribe/components/ConnectWalletButtons.tsx:62]: Lets keep the `Text` above, and just have the wallet button say "Connect wallet"

File: src/screens/subscribe/components/ConnectWalletButtons.tsx (hunk 1 above)
Comment (R-iskey) [reply src/screens/subscribe/components/ConnectWalletButtons.tsx]: The `ConnectButton` component is not providing onClick, but what's the problem with using `useEffect`?

**Comment by nivo33** [comment]
Sorry, closing this. We found it's causing a lot issues with other parnters that have RainbowKit installed for themselves or other `wagmi` solutions

**Pull request closed without merging**